### PR TITLE
feat: refactor

### PR DESCRIPTION
**Pull request by sandren**
This PR completes the following refactors:

- future proofs the React import style for TSX
- sorts imports according to packages and local code
- updates the ESLint, Prettier, PostCSS, and Tailwind configurations (no rule changes)
- replaces styled components (Emotion) with Tailwind
- adds the classnames package for easier styling with Tailwind
- creates a new `<Icon />` component

### CODE REVIEW

**Comment by baiirun** [comment]
There's some places where styles are off. I'll update this comment as I find them.

This should have a divider between the dropdown list items.
CleanShot 2023-02-16 at 17 12 33@2x
https://www.figma.com/file/iWKsYButqqKWd1bqeBrUUj/Geo---Web?node-id=3991%3A38293&t=JcbBVmD1ERN8y1gu-4

File: apps/web/styles/styles.css [hunk 1]
```diff
@@ -1,95 +1,85 @@
...
+  }
 }
 
-*,
-::before,
-::after {
-  border-width: 0;
-  border-style: solid;
+.animate-fade-in {
+  animation: fadeIn 0.15s ease-in-out`;
```
Comment (baiirun) [inline apps/web/styles/styles.css:82]: Where are we using this? Can we just define this in tailwind's config instead?

**Review by baiirun — COMMENTED** [review]
Just a couple small things

File: apps/web/styles/styles.css (hunk 1 above)
Comment (sandren) [reply apps/web/styles/styles.css:82]: We're using it in the `card.tsx` component. We can move the definition to the Tailwind config.

**Pull request merged**